### PR TITLE
[RHACS] Operator install instructions update for 3.67

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Includes updated instructions for Operator installation.

### CODE REVIEW

File: modules/install-acs-operator.adoc [hunk 1]
```diff
@@ -14,10 +14,23 @@ Using the OperatorHub provided with {ocp} is the easiest way to install {product
...
 . Enter *Advanced Cluster Security* into the *Filter by keyword* box to find the {product-title} Operator.
+
 . Select the *{product-title} Operator* to view the details page.
+
 . Read the information about the Operator and click *Install*.
-. On the *Install Operator* page, keep the default values for *Update channel*, *Installation mode*, *Installed namespace*, and *Approval strategy*.
+
+. On the *Install Operator* page:
+
+** keep the default value for *Installation mode* as *All namespacs on the cluster*.
```
Comment (IAmATeaPot418) [inline modules/install-acs-operator.adoc:26]: namespaces is spelled incorrect @gaurav-nelson

File: modules/install-acs-operator.adoc [hunk 2]
```diff
@@ -14,10 +14,23 @@ Using the OperatorHub provided with {ocp} is the easiest way to install {product
...
 . Enter *Advanced Cluster Security* into the *Filter by keyword* box to find the {product-title} Operator.
+
 . Select the *{product-title} Operator* to view the details page.
+
 . Read the information about the Operator and click *Install*.
-. On the *Install Operator* page, keep the default values for *Update channel*, *Installation mode*, *Installed namespace*, and *Approval strategy*.
+
+. On the *Install Operator* page:
+
+** keep the default value for *Installation mode* as *All namespaces on the cluster*.
```
Comment (kmccarron-rh) [inline modules/install-acs-operator.adoc:26]: Per ISG, capitalize the first words in a list.  s/keep/Keep

File: modules/install-acs-operator.adoc [hunk 3]
```diff
@@ -14,10 +14,23 @@ Using the OperatorHub provided with {ocp} is the easiest way to install {product
...
+** for *Installed namespace* choose a specific namespace in which to install the Operator. Red Hat recommends installing the {product-title} Operator in the *rhacs-operators* namespace.
```
Comment (kmccarron-rh) [inline modules/install-acs-operator.adoc:27]: For parallelism between bullets, consider starting bullets with the action: "Choose a specific..."

File: modules/install-central-operator.adoc [hunk 4]
```diff
@@ -19,13 +19,14 @@ When you install {product-title} for the first time, you must first install the
 .Procedure
 . On the {ocp} web console, navigate to the *Operators* -> *Installed Operators* page.
 . Select the {product-title} Operator from the list of installed Operators.
-. By default, {ocp} lists the project as `openshift-operator`. Select *Project: openshift-operator* -> *Create project*.
+. {ocp} lists the project as `rhacs-operator` if you have installed the Operator in the recommended namespace. Select *Project: rhacs-operator* -> *Create project*.
```
Comment (kmccarron-rh) [inline modules/install-central-operator.adoc:22]: This sentence was a little hard to read especially without haing a preview. Perhaps starting it with the "if you have installed....  would make it smoother? Or, consider adding a comma before the "if".

File: modules/install-acs-operator.adoc (hunk 2 above)
Comment (lpettyjo) [reply modules/install-acs-operator.adoc:26]: Fix throughout. There is another instance of this on line 28.

**Comment by kmccarron-rh** [comment]
Hi @gaurav-nelson - Some minor suggestions.

**Comment by gaurav-nelson** [comment]
@kmccarron-rh @lpettyjo Thank you reviewing this PR. I've made the suggested changes. Please review and merge.

File: modules/install-acs-operator.adoc [hunk 5]
```diff
@@ -14,10 +14,23 @@ Using the OperatorHub provided with {ocp} is the easiest way to install {product
...
+
+. On the *Install Operator* page:
+
+** Keep the default value for *Installation mode* as *All namespaces on the cluster*.
+** Choose a specific namespace in which to install the Operator for the *Installed namespace* field. Red Hat recommends installing the {product-title} Operator in the *rhacs-operators* namespace.
+** Select automatic or manual updates for *Approval strategy*.
++
+If you choose automatic updates, when a new version of the Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator.
++
+If you choose manual updates, when a newer version of the Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.
```
Comment (kmccarron-rh) [inline modules/install-acs-operator.adoc:32]: minor nit -   s/to have the Operator updated/to update the Operator/

File: modules/install-central-operator.adoc [hunk 6]
```diff
@@ -19,13 +19,14 @@ When you install {product-title} for the first time, you must first install the
 .Procedure
 . On the {ocp} web console, navigate to the *Operators* -> *Installed Operators* page.
 . Select the {product-title} Operator from the list of installed Operators.
-. By default, {ocp} lists the project as `openshift-operator`. Select *Project: openshift-operator* -> *Create project*.
+. If you have installed the Operator in the recommended namespace, {ocp} lists the project as `rhacs-operator`. Select *Project: rhacs-operator* -> *Create project*.
 +
 [WARNING]
 ====
-You must install the {product-title} `Central` custom resource in its own project and not in the default `openshift-operators` project.
+* If you have installed the Operator in some other namespace, {ocp} shows the name of that namespace rather than `rhacs-operator`.
```
Comment (kmccarron-rh) [inline modules/install-central-operator.adoc:26]: s/some other namespace/a different namespace/

**Review by kmccarron-rh — COMMENTED** [review]
I left a couple of suggestions, otherwise lgtm!

**Pull request merged**